Core/contentblockerrules.js

```diff
@@ -217,6 +217,10 @@
         originalFetch = window.fetch;
       }
       window.fetch = function() {
+        if (arguments.length == 0) {
```

Comment:
Thanks so much for this @samsymons! One question, is it possible for `arguments` to be null?

Reply:
Should also be triple = for JS best practice

Reply:
@SlayterDev Great catch, thank you!

@brindy Hmm, very good question... I can't find any documentation that suggests it can be null, and MDN [mentions that it's always available in typical function usage](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/arguments):

> The arguments object is a local variable available within all non-arrow functions.